tests/broker.cpp

```diff
@@ -228,7 +228,9 @@ TEST(broker, freeze)
 
 	// The broker is frozen
 	handler.on_request(message_container(broker_connect::KEY_CLIENTS, client_id, {"freeze"}), respond);
-	ASSERT_EQ(0u, messages.size());
+	ASSERT_EQ(1u, messages.size());
```

Comment:
You should probably check that the message is in fact an "ack" message here.

Reply:
Sure, make sence

Reply:
Acknowledgment should be tested